resolwe_bio/tests/processes/test_diff_expression.py

```diff
@@ -148,3 +148,121 @@ def test_edger(self):
         self.assertFields(diff_exp, 'species', 'Dictyostelium discoideum')
         self.assertFields(diff_exp, 'build', 'dd-05-2009')
         self.assertFields(diff_exp, 'feature_type', 'gene')
+
+    @tag_process('differentialexpression-dexseq')
+    def test_dexseq(self):
+        with self.preparation_stage():
+            # Mock upload featureCount process
+            process = Process.objects.create(
+                name='Upload featureCount mock process',
+                requirements={
+                    'expression-engine': 'jinja',
+                    'resources': {
+                        'network': True,
+                    },
+                    'executor': {
+                        'docker': {
+                            'image': 'resolwebio/base:ubuntu-18.04',
+                        },
+                    },
+                },
+                contributor=self.contributor,
+                data_name='{{ fc.file|default("?") }}',
+                type='data:expression:featurecounts',
+                flow_collection="sample",
+                input_schema=[
```

Comment:
The only input that is needed is `fc`. You can hardcode the rest in the `run` section of the process.